docs/deploy-cephfs.md

```diff
@@ -26,6 +26,20 @@ make image-cephcsi
 
 ## Configuration
 
+**NOTE:** To make CephFS CSI driver version >= 1.1.0 work with Ceph v14.2.2
+cluster (not deployed by rook), you need to add the following settings in the
+`mgr` section of the ceph.conf used by the Ceph manager daemon, and restart the
+Ceph manager daemon.
+
+```
```

Comment:
add bash to the markdown

Reply:
This is a section in the ceph.conf configuration file, and not a bash command/shell execution, is it still required?

Reply:
there is no way to highlight the general config file content in markdown. so with these changes, the configuration won't be highlighted